src/Joomlatools/Console/Command/Database/AbstractDatabase.php

```diff
@@ -87,7 +87,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
         else
         {
-            $this->target_db_prefix = '';
+            $this->target_db_prefix = $input->getOption('mysql_db_prefix') ?: $input->getOption('mysql-db-prefix');
```

Comment:
@thuebschmann The code inside this `else`  block is only run when the user has explicitly set a database name. If we're overriding the database, it wouldn't make sense to use a different `target_db_prefix`: it should simply be set to an empty string. 

This is also mentioned in the option's description: https://github.com/joomlatools/joomlatools-console/blob/aaa04096a21402e827dbc2b1ac7d73704445ec56/src/Joomlatools/Console/Command/Database/AbstractDatabase.php#L59

Reply:
you are righrt
then we need the table prefix option